### DISCUSSION TITLE
Grid layout is not shown in the notebook viewed as a WebAssembly link

### DISCUSSION THREAD

**Discussion by hopezh** [marimo-team/marimo#3309] [Q&A]
I did the following: 

1. create a **grid layout** with a few components in the "**app view mode**"
2. save the notebook
3. click "**share -> create WebAssembly link**"
4. paste the webassembly link in a **web browser**

However, the grid layout is now shown. 

May I ask how to save the grid layout so that it can be shown when the notebook is shown as a webassembly link?

**Comment by mscolnick** [comment]
@hopezh, the different layout types generate a file _next_ to notebook, for example `layouts/notebook.grid.json`. But since the playground doesn't let you pass around a full project with files (and instead only the notebook code), you can't use the layouts in the playground. maybe in the future we can support this.

for now, i'd recommend using the Community Cloud: https://marimo.io/sign-up which does support multiple files per project and different layout types like the grid.
